content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -6,19 +6,15 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-By adding a waypoint proxy to your traffic flow you can enable more of [Istio's features](/docs/concepts). Waypoints are configured using the {{< gloss "gateway api" >}}Kubernetes Gateway API{{< /gloss >}}.
+By adding a waypoint proxy to your traffic flow you can enable more of [Istio's features](/docs/concepts). Waypoints are configured using {{< gloss "gateway api" >}}Gateway API{{< /gloss >}}.
 
 {{< warning >}}
-Usage of VirtualService with the ambient data plane mode is considered Alpha. Mixing with Gateway API configuration is not supported, and will lead to undefined behavior.
-{{< /warning >}}
-
-{{< warning >}}
-`EnvoyFilter` is Istio's break-glass API for advanced configuration of Envoy proxies. Please note that *`EnvoyFilter` is not currently supported for any existing Istio version with waypoint proxies*. While it may be possible to use `EnvoyFilter` with waypoints in limited scenarios, its use is not supported, and is actively discouraged by the maintainers. The alpha API may break in future releases as it evolves. We expect official support will be provided at a later date.
+The VirtualService and EnvoyFilter APIs are not supported in waypoints. [Read more below](#unsupported-apis).
```

Comment:
A note, the wording of the warning was that desired by TOC, so I'm not sure we want to change this without running it by them. It was one I inserted after a TOC meeting after confirming the wording with them.

Reply:
The expository paragraph has moved to line 166.